app/src/main/java/com/example/erabook/data/firebasedb/UserDataRemote.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.erabook.data.firebasedb
+
+import com.google.firebase.Timestamp
+import com.google.gson.annotations.SerializedName
+import java.util.Date
+
+data class UserDataRemote(
+    @SerializedName("userUid") var userUid: String,
+    @SerializedName("userName") var userName: String,
+    @SerializedName("userEmail") var userEmail: String,
+//    @SerializedName("user_location") var userLocation: Coordinates?,
```

Comment:
when I implement the location feature I need these, I just commented out so that it won't cause me a problem when I need the data class userDataRemote to use it somewhere in the app same goes for other commented code I have below